src/app/main.cpp

```diff
@@ -209,7 +210,7 @@ int main( int argc, char **argv )
 
 #ifdef RELATIVE_PREFIX_PATH
   qputenv( "GDAL_DATA", QDir::toNativeSeparators( app.applicationDirPath() + "/../share/gdal" ).toLocal8Bit() );
-  const QString projPath( QDir::toNativeSeparators( app.applicationDirPath() + "/../share/proj" ) );
+  projPath( QDir::toNativeSeparators( app.applicationDirPath() + "/../share/proj" ) );
   app.setPrefixPath( app.applicationDirPath() + "/..", true );
 #else
   app.setPrefixPath( CMAKE_INSTALL_PREFIX, true );
```

Comment:
The important bit being it's empty in this `#else` branch?

Reply:
@m-kuhn , not following you here, what do you mean?

Reply:
Could we instead of the current approach (which fails some cppchecks) add a `const QString projPath;` at this position here to fix it?

Reply:
I think it'd still fail with cppcheck since it'd still be in the eyes of cppcheck an always empty string. That said, if you want to add a const QString there instead of the current approach, I can adapt.

Reply:
Done.